frontend/app/components/comment-form/comment-form.tsx

```diff
@@ -273,10 +279,13 @@ export class CommentForm extends Component<Props, State> {
   }
 
   onDrop(e: DragEvent) {
-    if (!this.props.user) {
+    const isAnonymous = this.props.user && isUserAnonymous(this.props.user);
```

Comment:
`this.props.user` can be `User` or `null`. `isUserAnonymous()` is already checking whether user is equal to `null` or not.

Reply:
I suppose `isUserAnonymous` is incorrect. Anonymous is user who logged as anonymous, not null.

Reply:
Oh, yes, sorry. You are right.